react/features/base/tracks/functions.js

```diff
@@ -125,6 +126,105 @@ export function createLocalTracksF(options = {}, firePermissionPromptIsShownEven
         }));
 }
 
+/**
+ * Returns an object containing a promise which resolves with the created tracks &
+ * the errors resulting from that process.
+ *
+ * @returns {Promise<JitsiLocalTrack>}
+ */
+export function createPrejoinTracks() {
```

Comment:
moved here implementation from conference.js, with the exception of the screen sharing handling